src/styles/_frontendmentor.scss

```diff
@@ -0,0 +1,29 @@
+.front-end-mentor-header{
+    width: 255px;
+    height: 140px;
+    font-family: $font-jost;
+    color: $white;
+    background-image: url('../assets/suggestions/desktop/background-header.png');
+    line-height: 30px;
+    filter: drop-shadow(0px 4px 4px rgba(0, 0, 0, 0.25));
+    border-radius: 10px;
+
+    span:nth-child(1) {
+        font-weight: bold;
```

Comment:
I would stick with the font-weights that are numbers because they're more explicit

Reply:
updated!